config/targets.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-env node */
+module.exports = {
+  browsers: [
+    'ie 9',
+    'last 1 Chrome versions',
```

Comment:
This is where you can set browser compatibility preferences, to minimize the number of Javascript to ES5 transformations.

More info here: https://guides.emberjs.com/v2.14.0/configuring-ember/build-targets/